cerberus-domain/src/main/java/com/nike/cerberus/domain/UserCredentials.java

```diff
@@ -17,24 +17,26 @@
 package com.nike.cerberus.domain;
 
 import java.util.Arrays;
+import lombok.Getter;
+import lombok.Setter;
 
 /** Represents the user's credentials sent during authentication. */
 public class UserCredentials {
 
-  private final String username;
+  @Getter @Setter private String username;
```

Comment:
I think UserCredentials should avoid using @Setter. It's intended to never change its state after init.

Reply:
made changes